tensorflow/core/grappler/optimizers/auto_mixed_precision_lists.h

```diff
@@ -375,7 +375,8 @@ class AutoMixedPrecisionListsMkl : public AutoMixedPrecisionLists {
                                      "DepthwiseConv2dNativeBackpropInput",
                                      "MatMul",
                                      "BatchMatMul",
-                                     "BatchMatMulV2"};
+                                     "BatchMatMulV2",
+                                     "Einsum"};
```

Comment:
Do we need to wrap this with `#ifndef DNNL_AARCH64_USE_ACL` too?

Reply:
Never mind. I made this change internally in the [merge commit](https://github.com/tensorflow/tensorflow/commit/3f56300c412b4b113c6a551ad1459035671900af).

Reply:
thanks @penpornk for your help.